### DISCUSSION TITLE
Cannot Sync Wallet

### DISCUSSION THREAD

**Discussion by gabrielchiaradia** [Chia-Network/chia-blockchain#5566] [Support]
I have a problem with W10 1.1.5 version...  i got the blockchain synced and farming but wallet stuck "not synced" with an height of 168062, i restart GUI, i stop and start wallet, but the wallet continues NOT SYNCED , it says 1 connection, but the height is not moving ... i got 307298 blocks now so i m synced on the blockchain. I got the farm ok, but the wallet is death.

**Comment by gabrielchiaradia** [comment]
Update. At the moment the wallet changed to syncing, but from 168062 goes to 6112... i dont know what happened, but it appears that the wallet were reseted.

**Comment by BrandtH22** [comment]
@CoderArgentina , the newest version(s) of chia has fixes in place to resolve this issue. Please download it from here: https://www.chia.net/downloads/

Since this issue has been resolved with the latest version(s) of chia we will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions, comments, or if the issue persists after an update.

```
Please note that upgrading major versions of chia generally requires the config.yaml to be recreated using these steps:
1. Stop chia
2. Rename the config or move it to a different directory
3. In CLI run chia init
4. Copy any custom settings from the old config to the new (plot drive directories, pool info, trusted node info, etc) - be careful here as yaml is very picky about formatting including those leading spaces
5. Start Chia
```

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
